### PR TITLE
Adding Varchar Column Length

### PR DESCRIPTION
**Pull request by kmj940101**
I have noticed that some of the columns are missing sizes. 
It is always good to follow consistency.

### CODE REVIEW

File: src/autoscaler/scalingengine/db/scalingengine.db.changelog.yml [hunk 1]
```diff
@@ -38,25 +38,25 @@ databaseChangeLog:
                     nullable: false
               - column:
                   name: reason
-                  type: varchar
+                  type: varchar(255)
```
Comment (boyang9527) [inline src/autoscaler/scalingengine/db/scalingengine.db.changelog.yml]: we probably will need larger size for reason, message and error

Comment (kmj940101) [reply src/autoscaler/scalingengine/db/scalingengine.db.changelog.yml]: @boyang9527 Thank you for your comment. How about changing the data type to a longer type such as TEXT?

screenshot from 2017-09-20 09-29-09

**Comment by kmj940101** [comment]
@rohitsharma04 Thank you for your review. Why do we need another changeset?
I did not add new columns to the existing table neither update the value.

**Comment by paltanmoy** [comment]
@kmj940101 It's an alteration of table structure, if you don't add this in a new changeset, Liquibase wont consider this changes (in existing deployments) assuming it has all changes in the databasechangelog table

**Comment by boyang9527** [comment]
LGTM

**Comment by kmj940101** [comment]
I’m closing this pull request because of inactivity. One of the reasons why I made this pull request was mysql needs to define a maximum size for varchar. I will submit another pull request soon.
Related Issue: https://github.com/cloudfoundry-incubator/app-autoscaler/issues/174

**Pull request closed without merging**